wallet/src/de/schildbach/wallet/util/WalletUtils.java

```diff
@@ -195,7 +197,12 @@ public static Wallet restoreWalletFromProtobuf(final InputStream is,
     public static Wallet restoreWalletFromSeed(final List<String> words,
                                                    final NetworkParameters expectedNetworkParameters) throws IOException {
         try {
-            final Wallet wallet = new Wallet(Constants.NETWORK_PARAMETERS, new KeyChainGroup(Constants.NETWORK_PARAMETERS, new DeterministicSeed(words, null,"", Constants.EARLIEST_HD_SEED_CREATION_TIME)));//new WalletProtobufSerializer().readWallet(is, true, null);
+            DeterministicSeed seed =  new DeterministicSeed(words, null,"", Constants.EARLIEST_HD_SEED_CREATION_TIME);
+            KeyChainGroup group = new KeyChainGroup(Constants.NETWORK_PARAMETERS, seed);
+
+            group.addAndActivateHDChain(new DeterministicKeyChain(seed, Constants.BIP44_PATH));
+
+            final Wallet wallet = new Wallet(Constants.NETWORK_PARAMETERS, group);//new WalletProtobufSerializer().readWallet(is, true, null);
```

Comment:
Can we remove the commented code?

Reply:
Yes and there other places containing commented code that needs to be cleaned up as well.